code/game/objects/items/reagent_containers/hypospray.dm

```diff
@@ -166,6 +121,72 @@
 
 	return TRUE
 
+///Always draws reagents on right click
+/obj/item/reagent_containers/hypospray/afterattack_alternate(atom/A, mob/living/user)
+	if(!istype(user))
+		return FALSE
+	if(!in_range(A, user) || !user.Adjacent(A)) //So we arent drawing reagent from a container behind a window
+		return FALSE
+	can_draw_reagent(A, user, TRUE)
+
+///If it's possible to draw from something. Will draw_blood() when targetting a carbon, or draw_reagent() when targetting a non-carbon
+///Returns TRUE if we successfully draw from something
```

Comment:
Do this as proper multiline autodoc or just make it one line. It doesn't really need to be this detailed.